### PR TITLE
Add Armstrong Number In C# Fixes #1954

### LINKED ISSUE #1954: Armstrong Number using C#
I would like implement the program to check if a number is an Armstrong number or not using C# as a part of GSSoC ' 20. Please assign this to me.
@SakshiShreya @asha15 @jainaman224 @MastersAbh @Vaibhavraj10

### PR DESCRIPTION
**Pull request by BishalKrChanda**
Fixes #1954 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added program to check whether a number is armstrong or not in C#

#### Languages Used:
- C#

#### Files Added:
- armstrong.cs

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Armstrong_Number/Armstrong.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+using System;  
+  public class Armstrong
```
Comment (jainaman224) [inline Armstrong_Number/Armstrong.cs]: Incorrect indentation.

File: Armstrong_Number/Armstrong.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+   {  
+     public static void Main(string[] args)
```
Comment (jainaman224) [inline Armstrong_Number/Armstrong.cs]: Use 4 space indentation.

File: Armstrong_Number/Armstrong.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
+using System;  
+  public class Armstrong
+   {  
+     public static void Main(string[] args)  
+      {
+       // initialising variables
+       int  n,r,sum=0,temp;
```
Comment (jainaman224) [inline Armstrong_Number/Armstrong.cs]: Add spaces around operators like `=`, `+`

File: Armstrong_Number/Armstrong.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
+using System;
```
Comment (jainaman224) [inline Armstrong_Number/Armstrong.cs]: Fix file name. Have a look at https://github.com/jainaman224/Algo_Ds_Notes/tree/master/Armstrong_Number

Comment (BishalKrChanda) [reply Armstrong_Number/Armstrong.cs]: I made the requested changes. Please check @jainaman224

**Comment by jainaman224** [comment]
Please include issue number.

File: Armstrong_Number/Armstrong_Number.cs [hunk 5]
```diff
@@ -0,0 +1,35 @@
+using System;
+    public class Armstrong
```
Comment (jainaman224) [inline Armstrong_Number/Armstrong_Number.cs]: Incorrect indentation

File: Armstrong_Number/Armstrong_Number.cs [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        public static void Main(string[] args)
+        {
+        // initialising variables
+        int  n, r, sum = 0, temp;
```
Comment (jainaman224) [inline Armstrong_Number/Armstrong_Number.cs]: Incorrect indentation.

File: Armstrong_Number/Armstrong_Number.cs [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        // running while loop to get the condition number
+        while(n > 0)      
+        {
+            r = n % 10;      
+            sum = sum + (r * r * r);      
+            n = n / 10;
+        }
+        // checking if the number found using the condition of finding armstrong number is equal to the given number
+        // printing the statement according to the result of the condition
+        if(temp==sum)
```
Comment (somya-kapoor) [inline Armstrong_Number/Armstrong_Number.cs]: Add a space around ==

File: Armstrong_Number/Armstrong_Number.cs [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+        }
+        // checking if the number found using the condition of finding armstrong number is equal to the given number
+        // printing the statement according to the result of the condition
+        if(temp==sum)      
+            Console.Write("The Entered Number is an Armstrong Number.");      
+        else      
+            Console.Write("The Entered number is not an Armstrong Number.");
+    }
+}  
+/*
```
Comment (somya-kapoor) [inline Armstrong_Number/Armstrong_Number.cs]: Leave a line before it

File: Armstrong_Number/Armstrong_Number.cs (hunk 7 above)
Comment (BishalKrChanda) [reply Armstrong_Number/Armstrong_Number.cs]: Please check

**Comment by somya-kapoor** [comment]
@BishalKrChanda Please squash the commits

**Comment by somya-kapoor** [comment]
@BishalKrChanda Are you working on it?

**Comment by BishalKrChanda** [comment]
> @BishalKrChanda Are you working on it?

Yes I am trying to figure out how to squash the commits, sorry for the delay @somya-kapoor

**Comment by BishalKrChanda** [comment]
@somya-kapoor I merged the commits yesterday. Please check.

**Pull request merged**